pkg/handlers/httpproxy/variables.go

```diff
@@ -76,9 +77,10 @@ func (HTTPProxyVariables) VariableSchema() clusterv1.VariableSchema {
 					Description: "HTTPS proxy value.",
 					Type:        "string",
 				},
-				"no": {
-					Description: "No Proxy list.",
-					Type:        "array",
+				"additionalNo": {
+					Description: "Additional No Proxy list that will be added to the automatically calculated " +
+						"values that will apply no_proxy configuration for cluster internal network.",
```

Comment:
Could you please add the default no proxy list that this will be appended to?

Reply:
Added.